lib/rpc_spec/MOBILE_API.xml

```diff
@@ -0,0 +1,8426 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
@joeygrover by [the SDL-0234 proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0234-proxy-rpc-generation.md) this XML will be included in the library as a Git submodule. It seems the adding a copy of this here could be excess.

Reply:
Yes it can likely be removed